tests/integration/questionnaire/test_questionnaire_list_collector.py

```diff
@@ -259,3 +259,12 @@ def test_cancel_text_displayed_on_edit_block_if_exists(self):
         self.get(change_link)
 
         self.assertInBody("Don’t need to change anything?")
+
+    def test_warning_text_displayed_on_remove_block_if_exists(self):
```

Comment:
I wonder if we need the "if_exists"? I don't believe you can even get to the remove block if the list item doesn't exists.

Reply:
This was copied from the preceding tests. I read it as meaning "if the 'warning' property exists".